api/src/utils/default-user.ts

```diff
@@ -0,0 +1,58 @@
+// TODO: audit this object to find out which properties need to be updated.
+import { type Prisma } from '@prisma/client';
+
+export const defaultUser: Omit<Prisma.userCreateInput, 'email'> = {
```

Comment:
Thought: We should use something like this for the seed script.

Reply:
The type checking? Could do.

Reply:
Oh. No, I just meant instead of having so much "fixture" around, we just re-use this.